CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
```

Comment:
Please, change commit message to `migrations: fix bootstrap on cluster rolling upgrade`

Reply:
Fixed